integration/tests/neon_evm/test_revision.py

```diff
@@ -1003,3 +1027,239 @@ def send_transaction_steps(holder_account, accounts):
         for acc in data_accounts:
             data_acc_revision_after = evm_loader.get_data_account_revision(acc)
             assert data_acc_revision_after == 3
+
+    def test_revision_revert_by_iterative_second_trx(
+        self,
+        evm_loader,
+        operator_keypair,
+        treasury_pool,
+        new_holder_acc,
+        holder_acc,
+        neon_api_client,
+        revision_revert_contract,
+        session_user,
+        sol_client,
+        sender_with_tokens,
+        environment,
+    ):
+        holder1 = holder_acc
+        holder2 = new_holder_acc
+        recipient = session_user
+        operator_balance_pubkey = evm_loader.get_operator_balance_pubkey(operator_keypair)
+        balance_account_revision = evm_loader.get_balance_account_revision(
+            revision_revert_contract.balance_account_address
+        )
+        amount = 100000
+        evm_loader.deposit_neon(operator_keypair, sender_with_tokens.eth_address, 3 * amount)
```

Comment:
doesn't  sender_with_tokens have enough neons?

Reply:
I added tokens to sender_with_tokens